tests/export/test_census_collate.py

```diff
@@ -0,0 +1,40 @@
+"""EDDT uses three census surveys. ACS PUMS, census PUMS, and the census decennial. 
+ACS PUMS data comes from the MDAT API. 
+The census decennial and census PUMS currently come from spreadsheets prepated by DCP population.
+These data are combined during the export process where files by geography-year are sent
+to digital ocean. These files should be as consistent as possible, using similar column names,
+column orders, etc. 
+"""
+
+from external_review.external_review_PUMS import save_PUMS
+
+save_kwargs = {
+    "eddt_category": "demographics",
+    "geography": "puma",
+    "test_data": True,
+}
+demos_2000 = save_PUMS(year="2000", **save_kwargs)
+demos_0812 = save_PUMS(year="0812", **save_kwargs)
+demos_1519 = save_PUMS(year="1519", **save_kwargs)
+
+
+def test_matching_column_orders_demographics_0812_1519():
+    assert (demos_0812.columns == demos_1519.columns).all()
+
+
+def test_matching_column_orders_demographics_2000_0812_1519():
```

Comment:
I really like those tests and we will need to do them again for the demographics "redo"

Reply:
ok nice, would love your input on future tests. I stopped testing my code when I was trying to hit our deadline for demographics last week and think that was a bad decision. I'm seeing that testing saves time even on short turnarounds

Reply:
Would also like to chat with you both on the tests, I think with Erica's data it should be a little easier to get the expected number of columns (each indicator should have 5 columns by race + 5 columns for the total population) so that might be a simple way to check out DO outputs